integration_tests/cli/validator-demo.sh

```diff
@@ -177,6 +190,107 @@ check_response "$result" "\"pid\": $pid"
+
+
+echo "node admin disables validator"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld tx validator disable-validator --from "$account" --yes")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+echo "Get a validator $address from disabled-validator query"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-validator --address=$address")
+check_response "$result" "\"approvals\": \[\]"
+check_response "$result" "\"disabledByNodeAdmin\": true"
+echo "$result"
+
+test_divider
+
+
+echo "node admin enables validator"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld tx validator enable-validator --from "$account" --yes")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+
+echo "Get a validator $address from disabled-validator query"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-validator --address="$address"")
+check_response "$result" "Not Found"
+echo "$result"
+
+
+test_divider
+
+
+echo "Alice proposes to disable validator $address"
+result=$(dcld tx validator propose-disable-validator --address="$address" --from alice --yes)
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+
+echo "Get all proposed validators to disable. $address in the list"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator all-proposed-disable-validators")
+check_response "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+echo "$result"
+
+test_divider
+
+
+echo "Get a proposed validator to disable $address"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator proposed-disable-validator --address="$address"")
+check_response "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+echo "$result"
+
+
+test_divider
+
+
+echo "Bob approves to disable validator $address"
+result=$(dcld tx validator approve-disable-validator --address="$address" --from bob --yes)
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+
+echo "Get all proposed validators to disable. $address not in the list"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator all-proposed-disable-validators")
+response_does_not_contain "$result" "\"address\": \"$address\""
+echo "$result"
+
+
+test_divider
+
+
+echo "Get a validator $address from disabled-validator query"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-validator --address="$address"")
+check_response "$result" "\"creator\": \"$alice_address\""
+check_response "$result" "\"disabledByNodeAdmin\": false"
+echo "$result"
+
+
+test_divider
+
+
+echo "Get $address"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator node --address=$address")
+check_response "$result" "\"jailed\": true"
+echo "$result"
+
+
+test_divider
+
 echo "PASSED"
 
```

Comment:
Please add the following tests:
1) A validator is disabled by Trustees; check that owner Node Admin can enable it
2) Trustees revoke Node Admin account; a validator is disabled by Trustees; check that owner Node Admin can not enable it

Reply:
Done! ✅